spec/menu_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'menu'
+require 'order'
+
+describe Menu do
```

Comment:
Do not test against the return value _type_.  We are interested in what an object _does_ not what it _is_.